unitylibs/core/workflow/workflow-acrobat/action-binder.js

```diff
@@ -44,6 +44,40 @@ class ServiceHandler {
     }
   }
 
+  async fetchFromServiceWithRetry(url, options, timeLapsed=0, maxRetryDelay=120) {
+    try {
+      const response = await fetch(url, options);
+      const error = new Error();
+      const contentLength = response.headers.get('Content-Length');
+      if (response.status !== 200 && response.status !== 202) {
+        if (contentLength !== '0') {
+          const resJson = await response.json();
+          ['quotaexceeded', 'notentitled'].forEach((errorMessage) => {
+            if (resJson.reason?.includes(errorMessage)) error.message = errorMessage;
+          });
+        }
```

Comment:
I don't believe this is needed here. This was used in `fetchFromService` function to handle potential errors arising from `/asset` call. May just want to return the json here.